pkgs/by-name/li/linalg/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  pname = "lialg";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "sgorsten";
+    repo = "linalg";
+    tag = "v${version}";
```

nit: finalAttrs is generally preferred:
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "lialg";
  version = "2.2";

  src = fetchFromGitHub {
    owner = "sgorsten";
    repo = "linalg";
    tag = "v${finalAttrs.version}";
```